### ISSUE TITLE
ColorProto does not work on Integers

### ISSUE THREAD

**Issue by yuhan0** [Clojure2D/clojure2d#13]
The documentation states that a color can be represented by
> Integer - packed ARGB value. Example: 0xffaa01.

But the protocol only implements this for Long values: https://github.com/Clojure2D/clojure2d/blob/5262658d197035324333c9d53e1e5014e779f7ac/src/clojure2d/color.clj#L404

This means that eg. mapping clojure2d.color functions over `int[]` arrays throws an error:
```clojure
(map clojure2d.color/hue (int-array [0xffaa01]))
;; => No implementation of method: :to-color of protocol: #'clojure2d.color/ColorProto found for class: java.lang.Integer
```
Is this an oversight or are we expected to manually cast values to `long`?

**Comment by genmeblog** [comment]
It's an oversight. I'll deploy a snapshot today. Thanks!

**Comment by genmeblog** [comment]
Fixed in 1.2.0-SNAPSHOT
